File: C/palindrome.c
#include <stdio.h>
int main()
{
    int n, sum=0, r=0, i=1,c;
    printf("Enter a number: ");
    scanf("%d", &n);
    c=n;
    while(n>0)
    {
        r=n%10;
        sum=(sum*10)+r;
        n=n/10;
    }
    if (c==sum)
        printf("It is a palindrome.");
    else
        printf("It is not a palindrome.");
    return 0;
}

File: Python/palindrome.py
n=input("Enter a number : ")
a=0
x=n
while n>0:
    r=n%10
    a=(a*10)+r
    n=(n/10)
if x==a:
    print ("The given number is a palindrome")
else:
    print ("The given number is not a palindrome")
